sql/views.sql

```diff
@@ -69,6 +69,23 @@ CREATE OR REPLACE VIEW timescaledb_information.policy_stats as
     INNER JOIN _timescaledb_internal.bgw_job_stat js on j.id = js.job_id
   ORDER BY ht.schema_name, ht.table_name;
 
+-- view for background worker jobs
+CREATE OR REPLACE VIEW timescaledb_information.jobs AS
```

Comment:
Should this be named policies instead? We now have 2 views related to jobs: one is called jobs and the other is called policy stats. Since the primary use case is for setting up policies, I suggest we rename this as timescaledb_information.policies.

Reply:
the distinction i tried to introduce between job/policy is that job is something that is handled by the scheduler and it doesnt matter what it does, while policies are concrete implementations that have a specific purpose, so in that sense both should use job since the have nothing policy-specific in them

Reply:
then policy_stats should be renamed to job_stats. Also better to get wider input on the name so that we don't have to back and revise it.

Reply:
i would prefer job then it would also follow our catalog names, but keeping that job/policy distinction in mind i guess the new feature could also be called custom policies